### PR TITLE
added mac stuff to gitignore

### PR DESCRIPTION
**Pull request by jwishnie**
CRITICAL IMPROVEMENTS!! ;-)

### CODE REVIEW

File: gitconfig [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [user]
-	name = David McCann
-	email = david.a.mccann@gmail.com 
+	name = Jeff Wishnie
```
Comment (thedavidmccann) [inline gitconfig]: Any thoughts as to how to cleanly make this value parameterizable? Not exactly fork-friendly if you have to remember to change this back whenever you make a PR.

**Comment by jwishnie** [comment]
Good point... forgot to revert that before making the request... Not sure... Maybe split into too configs? The one to check into Quickstart and another included in that one just to hold name, email etc...

Using the 'include' config directive:

```
[include]
    path = /path/to/file
```

**Comment by thedavidmccann** [comment]
If you revert the change I can merge this PR, and then yes I think that makes sense! Given go.sh already has a prompt for email, I can just also add a prompt for name and emit both into a file that doesn't live in the repo.

**Comment by thedavidmccann** [comment]
Thanks!

**Pull request merged**